postcss.config.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  plugins: [
+    require('autoprefixer')({
+      grid: true
```

Comment:
m... ie11 support?

Reply:
grid: true? 
I don't know whether it helps, I didn't check